docs/application/web/get-started/overview.md

```diff
@@ -1,40 +1,42 @@
 # Get Started with Web Applications
 
-When creating your first Tizen Web application, you can select from the following options, which all use the Web-native languages, such as HTML5, CSS, and JavaScript:
+Tizen Web applications are created using the web-native languages, such as HTML, JavaScript, and CSS that are combined in a package.
```

Comment:
```suggestion
Tizen Web applications are created using not only well-known Web programming languages, such as HTML, CSS, and JavaScript but also Tizen Web API, based on JavaScript, that interacts with the Tizen native subsystems. 
```

If you think the suggestion is a little wordy, you can change it.

Reply:
Based on your suggestion in https://github.com/Samsung/tizen-docs/pull/1130#discussion_r444121718, and to maintain consistency across all the get started pages, I tried to tweak your suggestion a little:

You can create Tizen Web applications in the web-native languages such as HTML, JavaScript, and CSS. You can also use the Tizen Web APIs that interact with the Tizen native subsystems. To create your first Tizen Web application, you can select from the following profiles:

Please verify and share your valuable feedback.

Reply:
I thought 'web-native languages' expression was awkward so I left suggestion. I know it came from the original one.
In my point of view, there is no problem you rephrase this sentence except for those words.